Program.cs

```diff
@@ -1,7 +1,40 @@
 ﻿namespace Training {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
-}
\ No newline at end of file
+   internal class Program {
+      static void Main (string[] args) {
+         Console.Write ("Enter Password: ");
+         string Password = Console.ReadLine ();
+         if (check (Password) == "true") Console.WriteLine ("Strong password.");
+         else Console.WriteLine ("Not a strong password. \n" + check (Password)) ;
+      }
+      static string check (string pass) {
+         string lower = "", upper = "", digit = "", splchar = "", notstrong = "";
+         foreach (char c in pass) {
+            if (char.IsLower (c))
+               lower = "true";
+         }
+         foreach (char c in pass) {
+            if (char.IsUpper (c))
+               upper = "true";
+         }
+         foreach (char c in pass) {
+            if (char.IsDigit (c))
+               digit = "true";
+         }
+         foreach (char c in pass) {
+            string spl = " !@#$%^&*()-+";

```

Comment:
Space is not a valid character for password.

Reply:
Space removed.